pkg/provisioner/ironic/ironic.go

```diff
@@ -115,7 +115,7 @@ func init() {
 	}
 	ironicTrustedCAFile = os.Getenv("IRONIC_CACERT_FILE")
 	if ironicTrustedCAFile == "" {
-		ironicTrustedCAFile = "/opt/metal3/certs/ca/crt"
+		ironicTrustedCAFile = "/opt/metal3/certs/ca/tls.crt"
```

Comment:
Why name change? This breaks a pattern for other `/opt/metal3/certs/*` files

Reply:
The secret which is generated by cert-manager and when we mount the secret to the container the CA file is tls.crt and we cannot change that with cert-manager. So we need to change the `ironicTrustedCAFile` to `"/opt/metal3/certs/ca/tls.crt"`.   can you please explain a bit more, what will be broken, if the name is changed?

Reply:
You can set custom paths using ENV

Reply:
Not broken, please see other names of files related to certs, naming is different, this will look not pretty

Reply:
ironicClientCertFile = "/opt/metal3/certs/client/crt"

ironicClientPrivKeyFile = "/opt/metal3/certs/client/key"

Reply:
My point is, if we change naming scheme for one of this, lets make this uniform for all of them

Reply:
I think I got your point. will update the PR and have the same naming change in other places.